GitCommands/PathUtil.cs

```diff
@@ -188,36 +156,9 @@ public static bool IsValidPath(string aPath)
             return fi != null;
         }
                 
-        public static bool PathExists(string aPath)
```

Comment:
These methods was created to return false for cases when the given path is of incorrect form (causing exceptions being caught - lines 198-200). The uses case is for expanding to a full path based on environment PATH variable. We are seeking only for valid paths and don't want to propagate exceptions to UI.